mozci/task.py

```diff
@@ -415,6 +434,38 @@ def backfill(self, push):
         logger.info("Backfilling task '{}'".format(self.tags.get("label", "")))
         return self._trigger_action(backfill_action, hook_payload)
 
+    def compare_failure(self, other_task: Task):
+        # TODO
+        return self.is_potential_regression and other_task.is_potential_regression
+
+    def get_past_regressions(
```

Comment:
Some of the logic here is redundant with https://github.com/mozilla/mozci/blob/e965ac88a9bd27123bc5cd8428daa717d6785f25/mozci/push.py#L984. You could have a quick check to see if that matches what we need (by calling `get_regressions("label")`).
Don't spend too much time on it though, we can just add a new function and then merge the two later once things stabilize.

Reply:
Looking at the code, it probably doesn't work as-is because it assumes tests and not builds, so there are a few things to fix to make the function also support builds.

Again, you can implement the logic in a separate function and then we merge them later, or you can try implementing it in the same function